store/sqlstore/team_store.go

```diff
@@ -209,10 +209,12 @@ func newSqlTeamStore(sqlStore SqlStore) store.TeamStore {
 		table.ColMap("DisplayName").SetMaxSize(64)
 		table.ColMap("Name").SetMaxSize(64).SetUnique(true)
 		table.ColMap("Description").SetMaxSize(255)
+		table.ColMap("Type").SetMaxSize(255)
```

Comment:
Why 255?

Reply:
255 is the limit [we have in gorp](https://github.com/mattermost/gorp/pull/5/files#diff-da7e85db8d7b43216497bfee21f74cd2R89) for MySQL before converting the field into text which is not recommended to use as an index

Reply:
:+1: